eth/polls/polllogs.go

```diff
@@ -98,7 +98,7 @@ func (p *EthLogPoll) AllLogs(state loomchain.ReadOnlyState,
 	}
 
 	if end-start > p.maxBlockRange {
-		return nil, fmt.Errorf("max allowed block range (%d) exceeded", p.maxBlockRange)
+		start = end - p.maxBlockRange
```

Comment:
I think it's adjusting the wrong end of the range, if you adjust the start it means the polling will miss blocks.

Reply:
Actually this is fine as is, eth_getFilterLogs is not supposed to keep track of the last block read, it just re-evaluates the filter each time it's called.